internal/cmd/pipeline/command_update.go

```diff
@@ -31,11 +31,20 @@ func (c *command) newUpdateCommand(enableSourceCode bool) *cobra.Command {
 				Text: `Revoke privilege to activate Stream Designer pipeline "pipe-12345".`,
 				Code: `confluent pipeline update pipe-12345 --activation-privilege false`,
 			},
+			examples.Example{
+				Text: `Request to update Stream Designer pipeline "pipe-12345", with new ksql cluster Id.`,
+				Code: `confluent pipeline update pipe-12345 --ksql-cluster lksqlc-123456.`,
+			},
+			examples.Example{
+				Text: `Request to update Stream Designer pipeline "pipe-12345", with new schema registry cluster Id.`,
+				Code: `confluent pipeline update pipe-12345 --update-schema-registry`,
```

Comment:
Why are we passing the flag in this example if its default value is `true`? Passing this flag does nothing.